application/src/main/java/org/opentripplanner/standalone/config/DebugUiConfig.java

```diff
@@ -0,0 +1,107 @@
+package org.opentripplanner.standalone.config;
```

Comment:
Why don't we put this inside the debugui package inside the config package (same question for other configs)?

Reply:
I have no idea and have just followed the pattern that was there before.

I would like to separate refactoring from the feature development. If we can come to an agreement how the packages should be structured I'm happy to change it in a follow up PR.